debian/mariadb-server.mariadb.init

```diff
@@ -156,19 +176,41 @@ case "${1:-''}" in
     # Start MariaDB!
     /usr/bin/mariadbd-safe "${@:2}" 2>&1 >/dev/null | $ERR_LOGGER &
 
-    for i in $(seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}"); do
+    for _ in $(seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}")
+    do
       sleep 1
-      if mariadbd_status check_alive nowarn ; then break; fi
+      if mariadbd_status check_alive nowarn
+      then
+        break
+      fi
       log_progress_msg "."
     done
-    if mariadbd_status check_alive warn; then
+    if mariadbd_status check_alive warn
+    then
       log_end_msg 0
       # Now start mysqlcheck or whatever the admin wants.
       output=$(/etc/mysql/debian-start)
-      if [ -n "$output" ]; then
+      if [ -n "$output" ]
+      then
         log_action_msg "$output"
       fi
     else
+      # Try one more time but save error log separately, then spit it out
+      # before logging ends and init script execution ends.
+      if pgrep -ax mariadbd > /dev/null
+      then
+        echo "ERROR: The mariadbd process is running but not responding:"
+        # shellcheck disable=SC2009
+        # Show the mariadbd process and it's parent and next line (if there is a child process)
+        ps faxu | grep mariadbd -C 1
+      else
+        ERROR_LOG_FILE="$(mktemp).err"
+        echo # ensure newline
+        timeout --kill-after=20 10 /usr/bin/mysqld_safe "${@:2}" --log-error="$ERROR_LOG_FILE"
```

Comment:
`mariadbd-safe`

Reply:
Ok with this one.. there is also file: `debian/additions/mariadb.conf.d/50-mysqld_safe.cnf` which is also installed..

Reply:
exe not changed yet.

Reply:
Please PR #3240 for this as it should take care of these ones

Reply:
I've also added them here as mentioned